From d32859bc1e57f8a127161f01543a79be0a159cb9 Mon Sep 17 00:00:00 2001
From: Philippe Gerum <rpm@xenomai.org>
Date: Tue, 31 Dec 2019 12:24:08 +0100
Subject: [PATCH 0583/1010] x86/build: evl: drop pointless include directives

These directives would also cause compilation errors in arch/x86/kvm
when building in-tree due to pulling the wrong header file
(i.e. "irq.h").

Signed-off-by: Philippe Gerum <rpm@xenomai.org>
---
 arch/x86/Makefile | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/x86/Makefile b/arch/x86/Makefile
index f75757344e..a4764ec923 100644
--- a/arch/x86/Makefile
+++ b/arch/x86/Makefile
@@ -190,10 +190,6 @@ KBUILD_CFLAGS += -Wno-sign-compare
 #
 KBUILD_CFLAGS += -fno-asynchronous-unwind-tables
 
-ifeq ($(CONFIG_EVL),y)
-KBUILD_CFLAGS += -Iarch/$(SRCARCH)/evl/include -Iinclude/evl
-endif
-
 # Avoid indirect branches in kernel to deal with Spectre
 ifdef CONFIG_RETPOLINE
   KBUILD_CFLAGS += $(RETPOLINE_CFLAGS)
-- 
2.30.2

